### PR TITLE
[37] Setting formatOnSave and fixing extension suggestion

### PR DESCRIPTION
**Pull request by lucis**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "deno.lint": true,
   "deno.unstable": true,
   "deno.importMap": "./import_map.json",
+  "[typescript]": { "editor.defaultFormatter": "denoland.vscode-deno" },
```
Comment (lucis) [inline .vscode/settings.json:6]: https://deno.land/manual@v1.25.4/tools/formatter

File: .vscode/extensions.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "recommendations": [
     "ecmel.vscode-html-css",
-    "bradlc.vscode-tailwindcss",
+    "sastan.twind-intellisense",
```
Comment (lucis) [inline .vscode/extensions.json:4]: https://deno.com/blog/fresh-1.1#official-twind-plugin

File: .vscode/settings.json (hunk 1 above)
Comment (igorbrasileiro) [reply .vscode/settings.json:6]: This typescript entry works for tsx files?

**Review by igorbrasileiro — APPROVED** [review]

**Comment by lucis** [comment]
@igorbrasileiro for some reason, I can't reply there, but that's right. 

I went to check opening a `.tsx` file, clicking on `Typescript React` in the bottom right, and clicking on "Configure something for this file"

File: .vscode/settings.json (hunk 1 above)
Comment (lucis) [reply .vscode/settings.json:6]: [This one](https://github.com/deco-pages/start/pull/9/files#diff-a5de3e5871ffcc383a2294845bd3df25d3eeff6c29ad46e3a396577c413bf357R7) does

**Review by igorbrasileiro — APPROVED** [review]

**Pull request merged**